tests/e2e/specs/admin-tracking.test.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, deactivatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+import { URL } from 'url';
+import { resetSiteKit } from '../utils';
+
+describe( 'Providing client configuration', () => {
+
+	beforeAll( async() => {
+		await activatePlugin( 'google-site-kit' );
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+		await resetSiteKit();
+	} );
+
+	beforeEach( async() => {
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-settings' );
+		await page.waitForSelector( '.mdc-tab-scroller__scroll-content' );
+		page.click( '.mdc-tab-scroller__scroll-content button:last-child' );
```

Comment:
Without going through this manually, one has no idea what this clicks on.

Let's use the same approach as in #343 for this for consistency reasons:

```js
await visitAdminPage( 'admin.php', 'page=googlesitekit-settings' );
await page.waitForSelector( '.mdc-tab-scroller__scroll-content' );

// Click on Admin Settings Tab.
await expect( page ).toClick( 'button.mdc-tab', { text: 'Admin Settings' } );
```

Reply:
Thanks for the consistency! This is updated.